gatsby_fp/src/components/fieldComponents/Education.js

```diff
@@ -15,20 +15,15 @@ export default function Education({ educationList }) {
               dangerouslySetInnerHTML={{
                 __html: ` 
                ${
-                 (element.deg && !(element.degother))
+                 (element.deg && !element.degother)
                    ? `${element.deg}`
                    : ``
                }
                ${
-                (element.degother && !(element.deg))
+                (element.degother && element.deg !== "Other")
```

Comment:
To allow for the possibility that the string "Other" becomes something else in the future such as "Other (please specify)" we should avoid using the string in our logic. Instead, we can just go ahead and render `degother` whenever it exists. If it turns out that for some reason the DM interface actually lets people populate `degother` even when `deg` isn't "Other", then we can go back to this as a last resort.